vars/util.groovy

```diff
@@ -194,4 +194,15 @@ GIT INFORMATION REPORT
     } else {
         println '[WARNING] The variable GIT_INFORMATION_REPORT does not exist'
     }
+}
+
+def getNextSnapshot(String version)
+{
+    String delimeter= "\\.\\s*"
+    String [] xyz = version.split(delimeter)
+    int z = Integers.parseInt(xyz[2])
+    z = z + 1
+    xyz[2] = z.toString()
+    String v = xyz[0] + "." + xyz[1] + "." + xyz[2] + "-snapshot"
+    return v
```

Comment:
A bit of changes:

```suggestion
    String [] versionSplit = version.split("\\.")
    if (versionSplit.length == 3) {
        int majorVersion = Integer.parseInt(versionSplit[0])
        int minorVersion = Integer.parseInt(versionSplit[1])
        int microVersion = Integer.parseInt(versionSplit[2])
        return "${majorVersion}.${minorVersion}.${microVersion+1}-SNAPSHOT"
    } else {
        error "Version is not in the format X.Y.z"
    }
```

=> more meaningful names and you also check that format is good and all values are integers

Reply:
Thsnkd fot this it's more appropriate way to do it.